public/index.html

```diff
@@ -1,89 +1,93 @@
+<!DOCTYPE html>
 <html>
-
 <head>
-  <style>
-    h1 {
-      margin: 4rem;
-    }
-
-    body, main {
-      text-align: center;
-    }
-
-    a {
-      color: white;
-      background-color: #9147FF;
-      text-decoration: none;
-      padding: 1rem;
-    }
-
-    .error {
-      display: none;
-    }
-  </style>
+    <title>Will It Blend?</title>
+    <style>
+        h1 {
+            margin: 4rem;
+        }
+
+        body, main {
+            text-align: center;
+        }
+
+        a.auth-button {
+            color: white;
+            background-color: #9147FF;
+            text-decoration: none;
+            padding: 1rem;
+        }
+
+        .hidden {
+            display: none;
+        }
+
+        .error-message {
+            color: red;
+        }
+    </style>
 </head>
-
 <body>
 
-  <h1>Will It Blend?</h1>
-
-  <noscript>
-
-    <h2>Error: JavaScript Required</h2>
-
-    <p>This page requires JavaScript to be able to send a request to the local server.</p>
-
-  </noscript>
-
-  <main hidden>
-
-    <a class="auth-button" href="https://id.twitch.tv/oauth2/authorize?client_id={{.ClientId}}&redirect_uri=http://localhost:{{.Port}}&response_type=token&scope={{.Scope}}">Login with Twitch</a>
-
-    <div class="success" hidden>
-      <h2>Successfully authorized via Twitch!</h2>
-      <p>The command will run. You may now close this page.</p>
-    </div>
-
-    <div class="error" hidden>
-      <h2>There has been an error logging into Twitch.</h2>
-    </div>
-
-  </main>
-
-  <script type="text/javascript">
-
-    document.querySelector('main').toggleAttribute("hidden");
-
-
-    async function handleHash() {
-      console.log("hash", location.hash)
-      const accessToken = location.hash.split("&")?.[0].replace("#access_token=", "");
-
-      if (!accessToken) {
-        console.log("No access token. Bailing out")
-        return;
-      }
-
-      const response = await fetch("http://localhost:{{.Port}}/auth", {
-        method: 'POST',
-        body: JSON.stringify({
-          token: accessToken
-        })
-      });
-
-      if (response.ok) {
-        document.querySelector('.auth-button').toggleAttribute("hidden");
-        document.querySelector('.success').toggleAttribute("hidden");
-      } else {
-        console.log("Error: Could not post auth.")
-        document.querySelector('.error').toggleAttribute("hidden");
-      }
-
-  }
-
-  handleHash();
-  </script>
-
+    <h1>Will It Blend?</h1>
+
+    <noscript>
+        <h2>Error: JavaScript Required</h2>
+        <p>This page requires JavaScript to be able to send a request to the local server.</p>
+    </noscript>
+
+    <main class="hidden">
```

Comment:
question: Why did this need to be changed to a class? Was there an issue with using `hidden`? It's a fairly standard html attribute.